README.md

```diff
@@ -1,14 +1,45 @@
-# Yuri-Ita
+# Yuri-ita
+
+🚧 This gem is in active development 🚧
+
+The yuri-ita (揺り板), Japanese for "rocking plate" is a traditional wooden gold
+pan used in Japan.
 
 The goal of this library is to provide developers with a powerful set of tools
-to build expressive user interfaces for filtering, searching, and sorting data.
+to build expressive user interfaces for filtering, searching, and sorting data
+in Ruby on Rails. It does not provide UI components but instead offers a simple
+interface that allows you to build your own UI.
 
-TODO: What problem are we solving
+[Check out a demo application showcasing some of the capabilities of the
+library.][demo]
 
-TODO: How does this library solve that problem
+[demo]: https://yuriita-prerelease.herokuapp.com/movies
 
-The yuri-ita (揺り板), Japanese for "rocking plate" is a traditional wooden gold
-pan used in Japan.
+## Introduction
+
+Yuri-ita offers a powerful set of tools to build powerful user interfaces that
+allow users to find exactly what they need while allowing designers and
+developers full control of the UI, filtering, and searching. Powerful primitives
+allow developers to compose the query and sorting behavior while using existing
+ActiveRecord scopes that already exist in your application.
+
+Each table is created from a flexible configuration of behaviors. Developers
```

Comment:
I don't think it's clear what we mean by 'table' here. It's not a database table or an html table. We should define this.

Reply:
Good call! Added a sentence in https://github.com/thoughtbot/yuri-ita/pull/66/commits/fc3b15ca90cd5a4e3463502cd3baefcb20bffe64 Is that clearer?

Reply:
Yes! Thank you